docs/buildstrategies.md

```diff
@@ -410,7 +410,27 @@ Contrary to the strategy `spec.parameters`, you can use system parameters and th
 | ------------------------------ | ----------- |
 | `$(params.shp-source-root)`    | The absolute path to the directory that contains the user's sources. |
 | `$(params.shp-source-context)` | The absolute path to the context directory of the user's sources. If the user specified no value for `spec.source.contextDir` in their `Build`, then this value will equal the value for `$(params.shp-source-root)`. Note that this directory is not guaranteed to exist at the time the container for your step is started, you can therefore not use this parameter as a step's working directory. |
-| `$(params.shp-output-image)`      | The URL of the image that the user wants to push as specified in the Build's `spec.output.image`, or the override from the BuildRun's `spec.output.image`. |
+| `$(params.shp-output-directory)` | The absolute path to a directory that the build strategy should store the image in. You can store a single tarball containing a single image, or an OCI image layout. |
+| `$(params.shp-output-image)`     | The URL of the image that the user wants to push as specified in the Build's `spec.output.image`, or the override from the BuildRun's `spec.output.image`. |
+| `$(parama.shp-output-insecure)`  | Flag that indicates if the output image's registry location is insecure, which means that it uses a certificated not signed by a certificate authority, or uses HTTP |
+
+### Output directory vs output image
+
+As a build strategy author, you decide whether you just build the image, or also push it to the container registry. You implicitly indicate how your strategy behaves by using, or not using the `$(params.shp-output-image)` system parameter:
+
+- If you DO NOT use `$(params.shp-output-image)`, then Shipwright assumes that your build strategy PUSHES the image. We call this a strategy-managed push.
+- If you DO use `$(params.shp-output-image)`, then Shipwright assumes that your build strategy does NOT PUSH the image. We call this a shipwright-managed push.
+
+When you use the `$(params.shp-output-image)` parameter, then Shipwright will also take care to set the [image-related system results](#system-results).
```

Comment:
```suggestion
When you use the `$(params.shp-output-image)` parameter, then Shipwright also takes care to set the [image-related system results](#system-results).
```
Can we use a more specific phrase than "takes care to set"? Does that mean Shipwright sets the system results, or it ensures that someone performs an action?

Reply:
Thanks. Changed it to

> When you use the `$(params.shp-output-image)` parameter, then Shipwright will also set the [image-related system results](#system-results).